### PR TITLE
 dependencies-upgrade

### PR DESCRIPTION
**Pull request by AminTi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,14 +34,15 @@
...
+    "@types/node": "20.4.9",
+    "ts-node": "^8.0.2",
+    "typescript": "4.4.3"
   },
   "dependencies": {
-    "azure-devops-node-api": "^6.5.0",
-    "guid-typescript": "~1.0.8",
-    "jsonc-parser": "~3.1.0",
-    "minimist": "~1.2.7",
+    "azure-devops-node-api": "6.6.0",
```
Comment (madkoo) [inline package.json:42]: Why a specifc version ?

File: src/nodejs/tsconfig.json [hunk 2]
```diff
@@ -2,8 +2,13 @@
   "compilerOptions": {
     "noImplicitAny": false,
     "noImplicitThis": true,
+<<<<<<< HEAD:src/NodeJs/tsconfig.json
```
Comment (madkoo) [inline src/nodejs/tsconfig.json:5]: This file sees messed up =:)

File: package.json (hunk 1 above)
Comment (madkoo) [reply package.json:42]: I would use ^  in front of all version to have consistency

**Review by madkoo — COMMENTED** [review]
Check comments

**Review by madkoo — COMMENTED** [review]
.DS_Store file should not be in the the repo, add a .gitignore file it is missing and make sure it is ignored :)

**Review by madkoo — COMMENTED** [review]
the .DS_Store file is still part of the PR, Please delete it from there

**Review by madkoo — APPROVED** [review]
Nice!

**Review by madkoo — APPROVED** [review]
Approved

**Review by madkoo — APPROVED** [review]
Approve again

**Comment by MOlausson** [comment]
Restarting work in new branch

**Pull request closed without merging**